src/tim/vx/platform/grpc/grpc_platform.cmake

```diff
@@ -0,0 +1,58 @@
+list(APPEND ${TARGET_NAME}_SRCS
```

Comment:
use CMakeLists.txt instead.

*.cmake should not used for source code but for module description in cmake conversion in my understanding, if you don't agree, please find some reference usage.